### ISSUE TITLE
Lost marks in Design Document

### ISSUE THREAD

**Issue by JordanToering** [Donnervoegel/java#219]
-2 SVN Contributions: Janice probably forgot to tell the TA about our github repo.
-1 No updated revision history: Its the initial release of the document.  I think we should argue this.
-4 Architecture Diagram: I feel like we should just show her the updated one that didn't get pushed properly.
-? Class Diagram: I don't think we can argue
-? Database Persistant: Didn't the image get cut off or is this the proper version?

Basically I think we were docked marks a lot of the time, not sure if all of it is valid.  Atleast the SVN stuff.  

Do we want to argue this or just leave it?
